src/kaitai_awkward/data/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 3.19...3.26)
+project(awkward_kaitai CXX)
+
+include(FetchContent)
+
+set(CMAKE_ENABLE_EXPORTS true)
+
+set(KAITAI_MAIN_FILE "" CACHE STRING "Specify the name of the main KSY file")
+set(KAITAI_STRUCT_CPP_STL_RUNTIME_VERSION "0.10.1")
+if(NOT DEFINED AWKWARD_VERSION)
+    message(FATAL_ERROR "AWKWARD_VERSION is not set")
+endif(NOT DEFINED AWKWARD_VERSION)
+
+if(NOT EXISTS "${KAITAI_MAIN_FILE}")
+   message(FATAL_ERROR "Could not find main file ${KAITAI_MAIN_FILE}")
+endif()
+
+# Parse Kaitai name
+cmake_path(GET KAITAI_MAIN_FILE PARENT_PATH KAITAI_DIR)
+cmake_path(GET KAITAI_MAIN_FILE STEM LAST_ONLY KAITAI_NAME)
+string(TOUPPER "${KAITAI_NAME}" KAITAI_NAME_UPPER)
+
+# Download kaitai struct headers
+FetchContent_Declare(
+  kaitai_struct_cpp_stl_runtime
+  URL      https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/archive/refs/tags/${KAITAI_STRUCT_CPP_STL_RUNTIME_VERSION}.zip
+  # Use patched CMakeLists.txt
+  PATCH_COMMAND ${CMAKE_COMMAND} -E copy
+        "${CMAKE_CURRENT_SOURCE_DIR}/kaitai_struct_cpp_stl_runtime/CMakeLists.txt" <SOURCE_DIR>/CMakeLists.txt
```

Comment:
Here is where we use CMake to patch (replace) the `kaitai-io/kaitai_struct_cpp_stl_runtime`'s `CMakeLists.txt` with our own.